### ISSUE TITLE
Add more details in error page - part 2

### ISSUE THREAD

**Issue by maxceem** [appirio-tech/connect-app#3047]
Recently we improved error page https://github.com/appirio-tech/connect-app/issues/2991

It works good when an error happens during loading of the project list. But to be able to show additional error data on the error page for all the cases we should do 2 things:
1. For each place where error happens, we should set in the redux store the whole error object instead of `true`/`false`. And we should make sure such changes wouldn't break existent code.

2. We should understand from where an error comes. And make sure that valuable error is saved in the redux store to be shown. For example error during dashboard loading https://github.com/appirio-tech/connect-app/commit/c8a3c9143a3d9a7c21304500db3cba282bfd323f#diff-a375321ae5cb9d8569b970755b938d30R29
     For the first view it's not clear from where this error will come, and would be valuable or no.

**Comment by vikasrohit** [comment]
@maxceem how much efforts do you think these 2 tasks would take? Also, I didn't understand the `2` point because if we have the error object, does it matter from where this error is raised? All we need is that stack trace.

**Comment by maxceem** [comment]
@vikasrohit to evaluate I have to do a little bit more research on this.

> Also, I didn't understand the 2 point because if we have the error object, does it matter from where this error is raised?

Maybe this is a false alarm but during some quick testing I didn't get an error object I've expected to get. But again, maybe I've just put `throw error` in a wrong place or something. Need to double check it, so mentioned it to not forget.

**Comment by vikasrohit** [comment]
Makes sense. So, I guess we should start on getting error details in other places as well as soon as possible may be via a separate f2f.

**Comment by vikasrohit** [comment]
@maxceem if possible, we need to add more details for more error situations as with our current implementation, the error details that we are showing to user, via the Copy Error details button, are no more than what they see on screen.
